lib/foods.js

```diff
@@ -135,6 +147,7 @@ function createNewFood(name, calories){
 $('document').ready(function() {
   // load the data
   reloadTables();
+  updateLineListener();
```

Comment:
@lucyconklin Can we move `updateLineListener()` into `reloadTables()`? That listener is only added when we load the page with any food items. Also we would have to make sure `updateLineListener()` is going to be refactored to avoid adding it to every single `td`. How to avoid memory leak topic from our JS class. 👍